yaml/roads.yaml

```diff
@@ -251,7 +251,7 @@ filters:
       kind_detail: {col: highway}
   - filter:
       highway: secondary
-    min_zoom: { clamp: { min: 0, max: 10, value: { call: { func: util.calculate_path_major_route, args: [ { col: fid } ] } } } }
```

Comment:
Context here is issues like: https://github.com/tilezen/vector-datasource/issues/1279